ios/Podfile

```diff
@@ -24,9 +24,11 @@ require_relative "../node_modules/react-native/scripts/react_native_pods"
 require_relative "../node_modules/@react-native-community/cli-platform-ios/native_modules"
 
 platform :ios, '12.4'
-# install! 'cocoapods', :deterministic_uuids => false
+install! 'cocoapods', :deterministic_uuids => false
```

Comment:
Nothing to do but I wonder why `:deterministic_uuids => false` is now the default 🤔